core/App/components/record/RecordDateIntField.tsx

```diff
@@ -0,0 +1,47 @@
+import dayjs from 'dayjs'
+import React, { useEffect, useState } from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+import { useTheme } from '../../contexts/theme'
+import { Attribute, Field } from '../../types/record'
+import { testIdWithKey } from '../../utils/testable'
+
+interface RecordBinaryFieldProps {
+  field: Field
+  shown?: boolean
+}
+const dateIntFormat = 'YYYYMMDD'
```

Comment:
Should this go in constants so that it can be changed in once place and impact the entire app wherever the format is used.